src/index.html

```diff
@@ -11,8 +11,67 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
According to [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), CSS files should be linked with a `.css` extension, not `.scss`. Please change the file extension in the href attribute.